src/limb_repo/environments/base_env.py

```diff
@@ -0,0 +1,38 @@
+"""Abstract Base Environment."""
+
+import abc
+
+from limb_repo.structs import LRState
+
+
+class BaseEnv(abc.ABC):
+    """An environment for the agent to interact with."""
+
+    def __init__(self) -> None:
+        pass
+
+    @abc.abstractmethod
+    def get_lr_state(self) -> LRState:
+        """Get the state of the active and passive arm."""
+
+    @abc.abstractmethod
+    def set_lr_state(self, state: LRState) -> None:
+        """Teleports the active and passive arm to the desired state.
+
+        *Does not step*.
+        """
+
+    # @abc.abstractmethod
```

Comment:
nit: remove any commented out code unless there's a very good reason to leave it